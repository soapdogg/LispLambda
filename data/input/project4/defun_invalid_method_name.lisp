(DEFUN PLUS () (MINUS 1 2))
(PLUS)