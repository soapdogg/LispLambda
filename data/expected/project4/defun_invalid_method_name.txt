Error! Invalid function name: PLUS